package.json

```diff
@@ -20,6 +20,13 @@
     "type": "git",
     "url": "git://github.com/wearereasonablepeople/monastic.git"
   },
+  "keywords": [
+    "monad",
+    "monadic",
+    "state",
+    "state management",
+    "functional"
+  ],
```

Comment:
You might add fantasyland or something. I imagine people would search by this tag.

Reply:
You're right I added it to Github but not to Npm.